### ISSUE TITLE
Cannot submit application to Apple Store / SDK 46

### ISSUE THREAD

**Issue by MylchenkoAlexandr** [expo/expo#21777]
### Summary

Today i try to submit some updates (app using SDK 46) and get this message:
P.S. Two weeks ago there were no problems!

**App Store Connect**
----
_Dear Developer, We identified one or more issues with a recent delivery for your app, "my app name" 1.0.4 (1). Your delivery was successful, but you may wish to correct the following issues in your next delivery:ITMS-90725: SDK Version Issue - This app was built with the iOS 15.5 SDK. All iOS apps submitted to the App Store must be built with the iOS 15 SDK or later, included in Xcode 13 or later.After you’ve corrected the issues, you can upload a new binary to App Store Connect._

_Best regards,
The App Store Team_

### What platform(s) does this occur on?

iOS

### SDK Version

46

### Environment

expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.2.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.16.0 - /usr/local/bin/node
      Yarn: 1.22.19 - /usr/local/bin/yarn
      npm: 8.11.0 - /usr/local/bin/npm
    IDEs:
      Xcode: /undefined - /usr/bin/xcodebuild
    npmPackages:
      @expo/metro-config: ^0.5.2 => 0.5.2 
      @expo/webpack-config: ^0.17.0 => 0.17.4 
      expo: ^46.0.0 => 46.0.20 
      react: 18.0.0 => 18.0.0 
      react-dom: 18.0.0 => 18.0.0 
      react-native: 0.69.6 => 0.69.6 
      react-native-web: ~0.18.7 => 0.18.12 
    Expo Workflow: managed

### Minimal reproducible example

NA

**Comment by MylchenkoAlexandr** [comment]
> I also saw something similar using Expo SDK 47. Unsure if anyone building with Expo 48 is seeing this issue yet.

Two week ago i already publish application to apple store (with SDK 48). Now my build crash for Android, and for iOS is allright.

**Comment by MylchenkoAlexandr** [comment]
> I also saw something similar using Expo SDK 47. Unsure if anyone building with Expo 48 is seeing this issue yet.

Anyone who use SDK 48 have other new problems... with application building... (see issues).

**Comment by Imtiajrex** [comment]
I managed to submit it using expo@48.0.0 but it failed when I tried with expo@47.0.13

**Comment by hongduclam** [comment]
I got the same error. Im using v47

**Comment by Imtiajrex** [comment]
> Last two weeks SDK 47 and SDK 48 have a many problems ,( I get a lot problems on my work. Something work for iOS, but not work for Android and vice versa. And not work for a both (like a crossplatform).

Thats true. Today I built for android it failed with the expo@48 had to go back to 47. Now need to check  if it works for ios or not.

**Comment by gabrielfurini** [comment]
Got the same warning using SDK 47, but just once. Two days after getting the error, without changing anything in my code nor upgrading/downgrading the Expo SDK, the warning was not shown anymore.
I suspect it was something in the Apple side. 🤔

**Comment by mighty-phoenix** [comment]
Facing the same issue on expo sdk 46

**Comment by bbenson09** [comment]
I received the same email and am using SDK 47. Note that it says "Your delivery was successful." It seems like the change is only going to go into effect starting in April, so submissions are still working.

My question for Expo is whether this means everyone on < Expo 47 will need to upgrade to 48 by April 1 if they wish to still submit to the App Store... 😬

**Comment by mighty-phoenix** [comment]
@bbenson09 upgrading the expo SDK is so painful sometimes!

**Comment by knorsen** [comment]
v47 is the oldest version to support iOS 16. From the Expo blog site "SDK 47 will be released in the fall with support for iOS 16." 
Weird that you also got that warning though @bbenson09 !

**Comment by mighty-phoenix** [comment]
I upgraded the expo sdk, set the latest images for ios in eas.json, and the problem is now solved for me..🎉🎉

**Comment by brentvatne** [comment]
Apple regularly bumps minimum supported versions of Xcode / iOS for submissions. We don't typically bump the default Xcode version used on EAS Build for already released Expo SDK versions, but in this case we bumped it to Xcode 14.1 from 14.0, which we believe to be a safe bump and will help many developers to avoid the issue in this thread. [You can always bump the Xcode version by setting the `image` on your build profile](https://docs.expo.dev/build-reference/infrastructure/#ios-server-images), but there is no guarantee that older SDK versions will compile with more recent versions of Xcode.
